src/Kernel32.Desktop/Kernel32+CreateFileFlags.cs

```diff
@@ -17,151 +17,158 @@ public partial class Kernel32
+            ///     in case, for file systems that support that naming. Use care when using this option, because files created with
+            ///     this flag may not be accessible by applications that are written for MS-DOS or 16-bit Windows.
             /// </summary>
-            PosixSemanticsFlag = 0x0100000,
+            FILE_FLAG_POSIX_SEMANTICS = 0x0100000,
 
             /// <summary>
-            /// Access is intended to be random. The system can use this as a hint to optimize file caching.
-            /// This flag has no effect if the file system does not support cached I/O and <see cref="NoBufferingFlag"/>.
-            /// For more information, see the Caching Behavior section of this topic.
+            ///     Access is intended to be random. The system can use this as a hint to optimize file caching. This flag has no
+            ///     effect if the file system does not support cached I/O and <see cref="FILE_FLAG_NO_BUFFERING" />. For more
+            ///     information, see the Caching Behavior section of this topic.
             /// </summary>
-            RandomAccessFlag = 0x10000000,
+            FILE_FLAG_RANDOM_ACCESS = 0x10000000,
 
             /// <summary>
-            /// The file or device is being opened with session awareness. If this flag is not specified, then per-session devices (such as a redirected USB device) cannot be opened by processes running in session 0. This flag has no effect for callers not in session 0. This flag is supported only on server editions of Windows.
+            ///     The file or device is being opened with session awareness. If this flag is not specified, then per-session
+            ///     devices (such as a redirected USB device) cannot be opened by processes running in session 0. This flag has no
+            ///     effect for callers not in session 0. This flag is supported only on server editions of Windows.
             /// </summary>
-            SessionAwareFlag = 0x00800000,
+            FILE_FLAG_SESSION_AWARE = 0x00800000,
 
             /// <summary>
-            /// Access is intended to be sequential from beginning to end. The system can use this as a hint to optimize file caching.
-            /// This flag should not be used if read-behind (that is, reverse scans) will be used.
-            /// This flag has no effect if the file system does not support cached I/O and <see cref="NoBufferingFlag"/>.
-            /// For more information, see the Caching Behavior section of this topic.
+            ///     Access is intended to be sequential from beginning to end. The system can use this as a hint to optimize file
+            ///     caching. This flag should not be used if read-behind (that is, reverse scans) will be used. This flag has no effect
+            ///     if the file system does not support cached I/O and <see cref="FILE_FLAG_NO_BUFFERING" />. For more information, see
+            ///     the Caching Behavior section of this topic.
```

Comment:
Yes it's Resharper with my current setting that is to indent nodes containing information that don't fill on one line (120 chars from start of comment if I remember well).

I did it on each file where I had to change comments for whatever reason to avoid generating hard to read long lines. And also for the ones that were really long already for me to be able to find the references I wanted to change without scrolling.

I prefer wrapped comment that are readable in the source code but if we want to standardize on anything about doc comments  we need a way to have some sort of auto cleanup out of R# as we can't expect all contributors to buy it.